doc/components.md

```diff
@@ -379,11 +379,19 @@ add(IsometricTileMapComponent(tileset, matrix));
 It also provides methods for converting coordinates so you can handle clicks, hovers, render
 entities on top of tiles, add a selector, etc.
 
-A more in-depth example can be found
-[here](https://github.com/flame-engine/flame/blob/main/examples/lib/stories/tile_maps/isometric_tile_map.dart).
+You can also specify the tileHeight, which is the vertical distance between the bottom and top
+planes of each cuboid in your tile. Basically, it's the height of the front-most edge of your
+cuboid; normally it's half (default) or a quarter of the tile size. On the image below you can see
+the height colored in the darker tone:
+
+![An example of how to determine the tileHeight](images/tile-height-example.png)
+
+This is an example of how a quarter-length map looks like:
 
 ![An example of a isometric map with selector](images/isometric.png)
 
+A more in-depth example, featuring how to parse coordinates to make a selector, can be found
+[here](https://github.com/flame-engine/flame/blob/main/examples/lib/stories/tile_maps/isometric_tile_map.dart).
```

Comment:
We could link the live here now

Reply:
I think the code is more illustrative of what someone looking at the docs want to see

Reply:
The example has a link to code, so but putting the example link here, we can have both? Or maybe we can have both links too.

Reply:
let me put both, links are free!